socorro/processor/signature_utilities.py

```diff
@@ -687,41 +687,32 @@ def _get_crashing_thread(self, processed_crash):
 
     #--------------------------------------------------------------------------
     def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
-        result = super(SignatureRunWatchDog, self)._action(
-            raw_crash,
-            raw_dumps,
-            processed_crash,
-            processor_meta
-        )
         processed_crash['signature'] = (
             "shutdownhang | %s" % processed_crash['signature']
         )
-        return result
+        return True
 
 
 #==============================================================================
-class SignatureShutdownTimeout(SignatureGenerationRule):
+class SignatureShutdownTimeout(Rule):
     """Customize the signature for shutdown timeouts"""
```

Comment:
We should expand this a bit. 
In particular, this doesn't "customize" the signature. It **overwrites the signature**. ...but only if the raw crash contains a truthy value for the key `AsyncShutdownTimeout`.